plugins/synaptics-cape/fu-synaptics-cape-device.c

```diff
@@ -289,7 +369,39 @@ fu_synaptics_cape_device_to_string(FuDevice *device, guint idt, GString *str)
 
 	g_return_if_fail(FU_IS_SYNAPTICS_CAPE_DEVICE(self));
 
-	fu_common_string_append_ku(str, idt, "active_partition", self->ActivePartition);
+	fu_common_string_append_ku(str, idt, "ActivePartition", self->ActivePartition);
+	fu_common_string_append_kb(str, idt, "InReportViaInterrupt", self->InReportViaInterrupt);
+}
+
+static gboolean
+fu_synaptics_cape_device_set_quirk_kv(FuDevice *device,
+				      const gchar *key,
+				      const gchar *value,
+				      GError **error)
+{
+	FuSynapticsCapeDevice *self = FU_SYNAPTICS_CAPE_DEVICE(device);
+	if (g_strcmp0(key, "InReportViaInterrupt") == 0) {
+		if ((g_strcmp0(value, "on") == 0) || (g_strcmp0(value, "true") == 0) ||
+		    (g_strcmp0(value, "yes") == 0) || (g_strcmp0(value, "1") == 0)) {
+			self->InReportViaInterrupt = TRUE;
+		} else if ((g_strcmp0(value, "off") == 0) || (g_strcmp0(value, "false") == 0) ||
+			   (g_strcmp0(value, "no") == 0) || (g_strcmp0(value, "0") == 0)) {
```

Comment:
If it's just one bit of information, I think registering a private flag is probably a better idea than using a quirk key-value. Then you can do something like `Flags = in-report-via-interrupt` in the quirk file.

Reply:
Thanks, will change it to flag instead.